aten/src/ATen/native/Resize.cpp

```diff
@@ -35,18 +35,21 @@ static bool is_functorch_wrapped_tensor(const Tensor& tensor) {
   return !(key_set & kFunctorchWrappedTensors).empty();
 }
 
-bool resize_output(const Tensor& output, IntArrayRef shape) {
+bool resize_output(const Tensor& output, IntArrayRef shape, c10::optional<IntArrayRef> strides) {
```

Comment:
`resize_output` is also used in a bunch of places outside of codegen, so maybe we should default strides to `c10::nullopt`?

Reply:
I also think so. The header file already does that!